scripts/wsrep_sst_xtrabackup-v2.sh

```diff
@@ -1476,7 +1475,24 @@ send_data_from_donor_to_joiner()
 
     pushd ${dir} 1>/dev/null
     set +e
-    timeit "$msg" "$strmcmd | $tcmd; RC=( "\${PIPESTATUS[@]}" )"
+
+
+    # Implement retry logic ourselves, as nc needs it, and socat has a bug
+    # when retry is used together with timeout
+    local rc=1
+    local counter=1
+
+    # remove the retry parameter from the command
+    local ltcmd=$(echo $tcmd | sed s/,sockopt=[0-9]*//)
```

Comment:
Are we going to remove all sockopt? not only retry=?
Something like:
`local ltcmd=${tcmd//,*retry=[0-9]*/""}`

Reply:
:facepalm: I wanted to write retry there instead of sockopt, fixed.